[FIX] 12.0 sale_invoice_group_method: Get correct invoice_group_method_id

Roodin opened the pull request:
Get the field from commercial partner if it is not set in partner

bguillot committed:
sale_promotion_rule: module to manage promotion rules

bguillot committed:
sale_promotion_rule: add button to clean promotion rule

lmignon committed:
sale_promotion_rule: Improve implementation
The module now supports multiple promotion rules.
When a coupon is used, it always take precedence on automatic promotion
rule.
The computation of promotions has been refactored to update lines from
the sale.order object. Sine lines are updated from the SO, the recompute
of SO fields that depends of lines is only tiggered at the end of the
update on the lines. (In place of each time a line is updated).
You can ask to recompute the promotion of a sale.order. If a coupon is
already specified it's conserved but all the automatic rules are applied
again.

lmignon committed:
[IMP] sale_promotion_rule: Remove demo data Avoid demo data with introducing a side effect on the expected results of all sale order created into unitest

lmignon committed:
[IMP] sale_promotion_rule: Improve README

lmignon committed:
[IMP] sale_promotion_rule: Supports discount amount
Before this change, only percentage were applicable as discount by
promotion rule.
This change allows you to define rules that apply a discount in the form of a specific amount. This amount is translated as a percentage disctributed on the different lines during the rule resolution in order to benefit from the standard mechanism for calculating discount by Odoo.

lmignon committed:
[IMP] sale_promotion_rule: s/restriction_amount/restriction_amount_field
The field reference the name of the field to use on the sale order to determine if the promotion is applicable according to the total amount.

lmignon committed:
[IMP] sale_promotion_rule: Overrides name_get in place of defining a new field for display_name

lmignon committed:
[IMP] sale_promotion_rule: Improve performance
When checking if a promotion rule can be applied according to its usage definition, usage search_count in place of search since we are not interested by the resultset

lmignon committed:
[FIX] sale_promotion_rule: We must also include Coupon into the multi rule strategy check

lmignon committed:
[IMP] sale_promotion_rule: Use discount product for discount defined with fixed amount
While the first approach based on the reuse of the standard Odoo field "discount_amount" seemed the least intrusive, it is unfortunately impossible to avoid rounding problems due to the transformation of a fixed amount into a percentage of dicount to be applied on the different products. Rounding problems are further aggravated when a so contains lines with products with a high price difference.
Since it's not possible to rely on the standard Odoo field, the implementation of the fixed amount discount is now based the usage of a dedicated product.

lmignon committed:
[IMP] sale_promotion_rule: Add hook to exclude lines from the sale order amount total
In some cases some lines must not be taken into account into the sale order total when checking if the minimal amount specified on the promotion rule is reached.
As side effect we must be more strict on the way we get the total on te sale order and if taxes are included or excluded into the total

lmignon committed:
[IMP] sale_promotion_rule: Improve promotion rule form

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

lmignon committed:
[IMP] sale_promotion_rule: Improve tests coverage

lmignon committed:
[FIX] sale_promotion_rule: Fix search for rules already used for one partner

Commit:
[FIX] prevent to aply an automatic rule on a sale promotion line

OCA-git-bot committed:
[ADD] icon.png

acsonefho committed:
[10.0][FIX] Fix sale_promotion_rule in case of multi rules

Commit:
[FIX] minimum amount must be evaluate without any discount applied

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_promotion_rule
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_promotion_rule/

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_promotion_rule
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_promotion_rule/

rousseldenis committed:
!fixup

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_promotion_rule 10.0.2.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_promotion_rule
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_promotion_rule/

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_promotion_rule
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_promotion_rule/

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

OCA-git-bot committed:
sale_promotion_rule 10.0.2.0.2

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_promotion_rule
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_promotion_rule/

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_promotion_rule
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_promotion_rule/

georginaf committed:
Added translation using Weblate (Catalan)

rousseldenis committed:
[10.0][FIX] sale_promotion_rule: Apply coupon to lines if percentage

If the coupon rule apply a discount in fixed price, the rule should
not be applied on each sale order line

Roodin committed:
[FIX] sale_invoice_group_method. Get correct invoice_group_method_id from commercial partner if not set in partner

Roodin committed:
[FIX] sale_invoice_group_method. Get correct invoice_group_method_id from commercial partner if not set in partner

pedrobaeza commented:
Take a look to https://github.com/OCA/account-invoicing/tree/12.0/sale_order_invoicing_grouping_criteria that is less invasive.

nikul-serpentcs commented on sale_invoice_group_method/model/partner.py:
```diff
@@ -11,3 +11,9 @@ class ResPartner(models.Model):
+    def _get_invoice_group_method_id(self):
+        if self.invoice_group_method_id:
+            return self.invoice_group_method_id
+        else:
```
```suggestion
        return self.commercial_partner_id.invoice_group_method_id
```

nikul-serpentcs commented on sale_invoice_group_method/model/partner.py:
```diff
@@ -11,3 +11,9 @@ class ResPartner(models.Model):
+        if self.invoice_group_method_id:
+            return self.invoice_group_method_id
+        else:
+            return self.commercial_partner_id.invoice_group_method_id
```
@Roodin no need to else part

pedrobaeza replied:
Both are needed, but a compact way to put it is:
```python
return self.invoice_group_method_id or self.commercial_partner_id.invoice_group_method_id
```

nikul-serpentcs replied:
@pedrobaeza  I mean method should be
```
def _get_invoice_group_method_id(self):
    if self.invoice_group_method_id:
        return self.invoice_group_method_id
    return self.commercial_partner_id.invoice_group_method_id
```

pedrobaeza replied:
And it's less good than my proposal as you can see.

Commit:
Apply black and 2to3

Commit:
[MIG] 12.0 sale_promotion_rule

victoralmau committed:
[IMP] sales_team_security: Auto-followers on main & invoice/shipping contacts + others

- For avoiding access errors on this records that are related to sales flow
- Sync salesman on parent change

Co-Authored-By: Pedro M. Baeza <pedro.baeza@tecnativa.com>

marcelsavegnago commented:
Is there something strange about this PR or am I wrong?

![image](https://user-images.githubusercontent.com/3595132/108630514-e9424f80-7443-11eb-9244-295f2cc4d06f.png)

chienandalu committed:
[ADD] sale_product_classification: New module

Commit:
[ADD] sale_resource_booking: sell booking products

This module extends the functionality of sale and resource bookings to support
creating pending resource bookings automatically when confirming a sale order
and to allow you to generate those sale orders easily from a resource booking
type.

@Tecnativa TT28202

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_invoice_plan
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_invoice_plan/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_invoice_plan
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_invoice_plan/

OCA-git-bot committed:
Merge PR #955 into 12.0

Signed-off-by dreispt

oca-travis committed:
[UPD] Update sale_substate.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[ADD] setup.py

Commit:
Added translation using Weblate (French (France))

Commit:
Translated using Weblate (French (France))

Currently translated at 100.0% (154 of 154 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_blanket_order
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_blanket_order/fr_FR/

sergiocorato committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_date
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_date/it/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (3 of 3 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_archive/pt/

eLBati committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_sequence
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_sequence/it/

sergiocorato committed:
[12.0][FIX] update hooked function to original function create_invoices
see https://github.com/odoo/odoo/commit/ff4f1eafd222555b6c13503a9d7253403899bcee

OCA-git-bot committed:
Merge PR #1605 into 12.0

Signed-off-by rousseldenis

oca-travis committed:
[UPD] Update sale_order_action_invoice_create_hook.pot

OCA-git-bot committed:
sale_order_action_invoice_create_hook 12.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_action_invoice_create_hook
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_action_invoice_create_hook/

oca-travis committed:
[UPD] Update sale_order_action_invoice_create_hook.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_action_invoice_create_hook
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_action_invoice_create_hook/

pedrobaeza committed:
[FIX] sales_team_security: Adjust security rules for team managers

A team manager wasn't able to access to their orders if the team was
different from their team.

Whole review has been done for the rest of the channel manager rules,
and added some tests.

Migration script included.

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (32 of 32 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_type
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_type/it/

Roodin committed:
Merge branch '12.0_FIX_group_from_commercial_partner' of github.com:Comunitea/sale-workflow into 12.0_FIX_group_from_commercial_partner

ypapouin committed:
Added translation using Weblate (French)

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (20 of 20 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_tag
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_tag/fr/

SilvioGregorini committed:
[FIX] sale_order_line_date: remove dependency to 'sale_stock' module

The module has no real need to depend on 'sale_stock' module, it only depends on fields and methods defined within 'sale' module.
Replacing 'sale_stock' with 'sale' will avoid installing an unused module.

OCA-git-bot committed:
Merge PR #1117 into 12.0

Signed-off-by rousseldenis

oca-travis committed:
[UPD] Update sale_promotion_rule.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_promotion_rule 12.0.1.0.1

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] addons table in README.md

bealdav committed:
Added translation using Weblate (French)

llacroix committed:
[FIX] use same version as other oca projects

This cause a conflict in some tools thinking they are different
versions.

    The conflict is caused by:
      The user requested openupgradelib 3.0.0 (from git+https://github.com/OCA/openupgradelib.git)
      The user requested openupgradelib 3.0.0 (from git+https://github.com/OCA/openupgradelib.git@master)

azucenatrey committed:
[FIX] sale_order_action_invoice_create_hook:

PROBLEM DETECTED:
    When the "sale_order_action_invoice_create_hook" module is installed, Odoo executes the internal function "new_action_invoice_create" instead of the generic "action_invoice_create", so the latter is not executed. So the functionality of the original function "action_invoice_create" and "new_action_invoice_create" cannot be inherited from another module because the latter function is private and is not known from outside.

PROPOSED SOLUTION:
    A new generic function "_modify_invoices" is added outside the hook and it is called from within "new_action_invoice_create" to be able to inherit it in other modules and to be able to make the necessary modifications.

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (30 of 30 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_price_history
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_price_history/it/

OCA-git-bot committed:
Merge PR #1636 into 12.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update sale_order_action_invoice_create_hook.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
sale_order_action_invoice_create_hook 12.0.1.0.3

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_action_invoice_create_hook
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_action_invoice_create_hook/

oca-travis committed:
[UPD] Update sale_order_action_invoice_create_hook.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_action_invoice_create_hook
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_action_invoice_create_hook/

rousseldenis committed:
[12.0][DEL] sale_order_min_qty: Remove trash folder

OCA-git-bot committed:
Merge PR #1640 into 12.0

Signed-off-by rousseldenis

OCA-git-bot committed:
Merge PR #1628 into 12.0

Signed-off-by dreispt

OCA-git-bot committed:
Merge PR #1072 into 12.0

Signed-off-by dreispt

OCA-git-bot committed:
product_form_sale_link 12.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #1617 into 12.0

Signed-off-by dreispt

OCA-git-bot committed:
sale_order_line_date 12.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_input
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_input/it/

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_global_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_global_discount/it/

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_discount_display_amount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_discount_display_amount/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_general_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_general_discount/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_triple_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_triple_discount/it/

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_fixed_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_fixed_discount/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_commercial_partner/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_partner_incoterm
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_partner_incoterm/it/

leemannd committed:
Fix sale_automatic_workflow: on some cases you ship more
products than planned. You want to consider them as fully shipped

manuelcalerosolis committed:
Add button to recalculate weight

sergio-teruel committed:
[ADD] sale_stock_last_date: New module
TT31039

OCA-git-bot committed:
Merge PR #1668 into 12.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update sale_stock_last_date.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #1659 into 12.0

Signed-off-by dreispt

OCA-git-bot committed:
sale_automatic_workflow 12.0.1.1.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #1557 into 12.0

Signed-off-by rousseldenis

oca-travis committed:
[UPD] Update sale_order_weight.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_order_weight 12.0.1.0.1

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #1546 into 12.0

Signed-off-by rousseldenis

OCA-git-bot committed:
sale_rental 12.0.1.1.1

OCA-git-bot committed:
[UPD] addons table in README.md

GSLabIt committed:
[12.0][IMP] sale_discount_display_amount, add discount amount on document report

stevech091 committed:
Translated using Weblate (Italian)

Currently translated at 28.5% (10 of 35 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_exception/it/

OCA-git-bot committed:
Merge PR #1681 into 12.0

Signed-off-by rousseldenis

oca-travis committed:
[UPD] Update sale_discount_display_amount.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_discount_display_amount 12.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_discount_display_amount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_discount_display_amount/

OCA-git-bot committed:
[UPD] addons table in README.md

eLBati committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_fixed_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_fixed_discount/it/

aaltinisik committed:
Translated using Weblate (Turkish)

Currently translated at 14.2% (2 of 14 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_revision
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_revision/tr/

sergiocorato committed:
[12.0][FIX] sale_order_line_date depends on sale_stock

OCA-git-bot committed:
Merge PR #1706 into 12.0

Signed-off-by tafaRU

OCA-git-bot committed:
sale_order_line_date 12.0.1.1.1

OCA-git-bot committed:
[UPD] addons table in README.md

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_sequence
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_sequence/fr/

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_quotation_number
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_quotation_number/fr/

yibudak committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (14 of 14 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_revision
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_revision/tr/

yibudak committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (35 of 35 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_exception/tr/

yibudak committed:
Translated using Weblate (Turkish)

Currently translated at 50.9% (28 of 55 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_automatic_workflow/tr/

sergiocorato committed:
Added translation using Weblate (Italian)

sergiocorato committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_stock_picking_note
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_stock_picking_note/it/

educasilva committed:
Translated using Weblate (Portuguese (Portugal))

Currently translated at 100.0% (55 of 55 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_automatic_workflow/pt_PT/

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 69.5% (16 of 23 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_cancel_reason/tr/

SicurSam committed:
Added translation using Weblate (Italian)

SicurSam committed:
Translated using Weblate (Italian)

Currently translated at 7.4% (4 of 54 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_restricted_qty/it/

OCA-git-bot committed:
Merge PR #1523 into 12.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update sale_resource_booking.pot

SicurSam committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (54 of 54 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_restricted_qty/it/

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 33.3% (1 of 3 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_action_invoice_create_hook
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_action_invoice_create_hook/it/

hbrunn committed:
[IMP] sale_force_whole_invoiceability: allow forcing in done state too; fix invisibility domain

OCA-git-bot committed:
Merge PR #1734 into 12.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
sale_force_whole_invoiceability 12.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

Commit:
Added translation using Weblate (Italian)

OCA-git-bot committed:
Merge PR #1520 into 12.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update sale_product_classification.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[ADD] setup.py

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (20 of 20 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_product_multi_add
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_product_multi_add/it/

eLBati committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (20 of 20 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_product_multi_add
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_product_multi_add/it/

jguenat committed:
Added translation using Weblate (French)

jguenat committed:
Added translation using Weblate (French (Switzerland))

jguenat committed:
Translated using Weblate (French)

Currently translated at 100.0% (42 of 42 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_resource_booking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_resource_booking/fr/

jguenat committed:
Translated using Weblate (French (Switzerland))

Currently translated at 100.0% (42 of 42 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_resource_booking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_resource_booking/fr_CH/

victoralmau committed:
[IMP] partner_contact_sale_info_propagation: Add context to prevent change partner propagation.
TT27986

OCA-git-bot committed:
Merge PR #1439 into 12.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update sales_team_security.pot

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sales_team_security
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sales_team_security/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (66 of 66 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_invoice_plan
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_invoice_plan/pt_BR/

victoralmau committed:
[FIX] sales_team_security: Rule domain corrections.
TT32673

OCA-git-bot committed:
Merge PR #1803 into 12.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #1265 into 12.0

Signed-off-by dreispt

oca-travis committed:
[UPD] Update sale_order_line_serial_unique.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

tack-yabeQuartile committed:
Added translation using Weblate (Japanese)

tack-yabeQuartile committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (10 of 10 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_delivery_state
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_delivery_state/ja/

toffax committed:
Added translation using Weblate (Italian)

toffax committed:
Added translation using Weblate (Italian)

toffax committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_delivery_state
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_delivery_state/it/

toffax committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (48 of 48 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_manual_delivery
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_manual_delivery/it/

OCA-git-bot committed:
Merge PR #749 into 12.0

Signed-off-by rousseldenis

oca-travis committed:
[UPD] Update sale_order_rename.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
sale_order_rename 12.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

sergio-teruel committed:
[IMP] sale_order_type: Add analytic account to order type

OCA-git-bot committed:
Merge PR #1841 into 12.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update sale_order_type.pot

OCA-git-bot committed:
sale_order_type 12.0.1.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_type
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_type/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_priority
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_priority/pt/

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_quotation_number
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_quotation_number/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_revision
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_revision/it/

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 76.3% (42 of 55 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_automatic_workflow/tr/

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 60.0% (6 of 10 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_automatic_workflow_payment_mode
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_automatic_workflow_payment_mode/tr/

yavuzavci committed:
Added translation using Weblate (Turkish)

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_mrp_link
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_mrp_link/tr/

yavuzavci committed:
Added translation using Weblate (Turkish)

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (1 of 1 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_disable_inventory_check
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_disable_inventory_check/tr/

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 33.3% (1 of 3 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_price_recalculation
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_price_recalculation/tr/

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_partner_incoterm
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_partner_incoterm/tr/

yavuzavci committed:
Translated using Weblate (Turkish)

Currently translated at 90.0% (9 of 10 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_validity
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_validity/tr/

Commit:
Added translation using Weblate (Italian)

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_digitized_signature
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_digitized_signature/it/

sysadminmatmoz committed:
Added translation using Weblate (Slovenian)

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 100.0% (3 of 3 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_stock_picking_note
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_stock_picking_note/sl/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_discount_display_amount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_discount_display_amount/it/

sergio-teruel committed:
[FIX] sale_product_set: Add invisible field to wizard view
TT34880

OCA-git-bot committed:
Merge PR #1935 into 12.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
sale_product_set 12.0.1.3.2

OCA-git-bot committed:
[UPD] addons table in README.md

sergiocorato committed:
Added translation using Weblate (Italian)

sergiocorato committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_invoice_policy
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_invoice_policy/it/

Commit:
Added translation using Weblate (French)

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (30 of 30 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_price_history
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_price_history/fr/

pedrobaeza committed:
[FIX] sale_global_discount: Missing dependency

OCA-git-bot committed:
[UPD] addons table in README.md

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_triple_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_triple_discount/it/

eugenichafer committed:
Translated using Weblate (Catalan)

Currently translated at 98.4% (65 of 66 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_promotion_rule
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_promotion_rule/ca/

Roodin committed:
Merge branch '12.0_FIX_group_from_commercial_partner' of github.com:Comunitea/sale-workflow into 12.0_FIX_group_from_commercial_partner
